src/assets/components/List.jsx

```diff
@@ -1,7 +1,7 @@
 import "./List.css";
 import InputBox from "./InputBox";
 import ListItem from "./ListItem";
-import React, { useState } from "react";
+import { useState } from "react";
```

Comment:
as you noticed in the video, `React` isn't required. the only import you need is:

```js
import { /* here the hooks you use */ } from 'react'
```

why do code generators (like chatGPT, or Copilot) include then this:

```js
import React from 'react'
```

well this was needed in prev. versions of React (I think React < v17), all the code generated before that version does include this import statement, that's why the generators include it. 

it was needed so the transpiler did know you were going to write some JSX syntax (if i recall correctly) but not needed anymore

TL;DR you can remove that safely